internal/server/admin/handler.go

```diff
@@ -323,7 +339,7 @@ func RegisterAdminService(server *grpc.Server, opts HandlerOpts) {
 	v1.RegisterTargetServiceServer(server, services.target)
 	v1.RegisterSchemasServiceServer(server, services.schemas)
 	v1.RegisterNodeServiceServer(server, services.node)
-	v1.RegisterStatusServiceServer(server, services.status)
+	v1.RegisterStatusServiceServer(server, services.statusV1)
```

Comment:
Don't we need to register `services.statusV2` as well?

Reply:
Good catch!

Reply:
Small change request so I force pushed the change.